en/chromebook_setup/instructions.md

```diff
@@ -61,6 +61,88 @@ This bottom area is your terminal. You can use the terminal to send instructions
 to the remote Cloud 9 computer. You can resize that window to make it a bit
 bigger.
 
+#### Glitch.com Cloud IDE
+
+1. Go to [Glitch.com](https://glitch.com/)
+2. Sign up for an account (https://glitch.com/signup) or use your GitHub account if you have one (see GitHub instructions below).
+3. Click _New Project_ and choose _hello-webpage_
+4. Click on the `Tools` dropdown list (at the bottom left side of the window), then on `Terminal` button to open terminal tab with a prompt like this:
+
+{% filename %}Terminal{% endfilename %}
+```
+app@name-of-your-glitch-project:~
+```
+
+When using Glitch.com as your Cloud IDE, we do not create a virtual environment but we create the following files manually :
+
+{% filename %}glitch.json{% endfilename %}
+```json
+{
+  "install": "pip3 install -r requirements.txt --user",
+  "start": "bash start.sh",
+  "watch": {
+    "throttle": 1000
+  }
+}
+```
+
+{% filename %}requirements.txt{% endfilename %}
+```
+Django~={{ book.django_version }}
+```
+
+{% filename %}.bash_profile{% endfilename %}
+```bash
+alias python=python3
+alias pip=pip3
+```
+
+
+{% filename %}start.sh{% endfilename %}
+```bash
+chmod 600 .bash_profile
+pip3 install -r requirements.txt --user
+python3 manage.py makemigrations
+python3 manage.py migrate
+python3 manage.py runserver $PORT
+```
+
+Once these files are created, open a terminal (in the tools menu at the bottom) and execute the following commands to create your first Django project :
+
+{% filename %}Terminal{% endfilename %}
+```
+django-admin.py startproject mysite .
+refresh
+```
+
+In order to see detailed error messages, you can activate Django debug logs for your Glitch application.
+Simply add the following at the end of the `mysite/settings.py` file.
+
+{% filename %}mysite/settings.py{% endfilename %}
+```python
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'handlers': {
+        'file': {
+            'level': 'DEBUG',
+            'class': 'logging.FileHandler',
+            'filename': 'debug.log',
+        },
+    },
+    'loggers': {
+        'django': {
+            'handlers': ['file'],
+            'level': 'DEBUG',
+            'propagate': True,
+        },
+    },
+}
+```
+This will create a `debug.log` file detailing Django operations and any error messages that might come up, making it much easier to fix if your website does not work.
+
+The initial restarting of the Glitch project should fail (if you click on the top dropdown button `Show` then click on `In a New Window`, you will receive a `DisallowedHost` error message). Do not worry about it at this stage, the tutorial will fix this as soon as you update the Django settings of your project in the `mysite/settings.py` file.
```

Comment:
The rest of your contribution has each sentence on a source line of its own, which, although most of the Django Girls tutorial isn't written that way, is IMO a good idea. (It's essentially the most basic form of ["semantic linefeeds"](https://rhodesmill.org/brandon/2012/one-sentence-per-line/) a.k.a. ["semantic line breaks"](https://sembr.org/).) Let's do apply that here, too.
```suggestion
The initial restarting of the Glitch project should fail (if you click on the top dropdown button `Show` then click on `In a New Window`, you will receive a `DisallowedHost` error message).
Do not worry about it at this stage, the tutorial will fix this as soon as you update the Django settings of your project in the `mysite/settings.py` file.
```

Maybe, let's also pull out the parenthesis from the sentence, as it contains a whole full sentence of its own.
```suggestion
The initial restarting of the Glitch project should fail.
(If you click on the top dropdown button `Show` then click on `In a New Window`, you will receive a `DisallowedHost` error message.)
Do not worry about it at this stage, the tutorial will fix this as soon as you update the Django settings of your project in the `mysite/settings.py` file.
```